geometry_msgs/Vector3 acceleration
geometry_msgs/Vector3 angular_velocity
geometry_msgs/Vector3 magnetic_field

float64 mouse_integrated_x
float64 mouse_integrated_y

float64 magnetic_field_strength
geometry_msgs/Vector4 est_orientation
